modules/nf-core/yahs/main.nf

```diff
@@ -0,0 +1,44 @@
+process YAHS {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda (params.enable_conda ? "bioconda::yahs=1.2a.2" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/yahs:1.2a.2--h7132678_0':
+        'quay.io/biocontainers/yahs:1.2a.2--h7132678_0' }"
+
+    input:
+    tuple val(meta), path(bed)
+    tuple path(ref), path(fai)
+    val(if_break)
```

Comment:
I think, according to [current guidelines](https://nf-co.re/docs/contributing/modules#general), that these should all go through `ext.args`.